tests/metrics/regression/test_psnr.py

```diff
@@ -0,0 +1,75 @@
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from skimage.metrics import peak_signal_noise_ratio
+import numpy as np
+
+from pytorch_lightning.metrics.regression import PSNR
+from pytorch_lightning.metrics.functional import psnr
+from tests.metrics.utils import BATCH_SIZE, NUM_BATCHES, MetricTester
+
+torch.manual_seed(42)
+
+
+Input = namedtuple('Input', ["preds", "target"])
+
+_inputs = [
+    Input(
+        preds=torch.randint(n_cls_pred, (NUM_BATCHES, BATCH_SIZE), dtype=torch.float),
+        target=torch.randint(n_cls_target, (NUM_BATCHES, BATCH_SIZE), dtype=torch.float),
+    )
+    for n_cls_pred, n_cls_target in [(10, 10), (5, 10), (10, 5)]
+]
+
+
+def _sk_metric(preds, target, data_range):
+    sk_preds = preds.view(-1).numpy()
+    sk_target = target.view(-1).numpy()
+    return peak_signal_noise_ratio(sk_target, sk_preds, data_range=data_range)
+
+
+def _base_e_sk_metric(preds, target, data_range):
```

Comment:
what is the `_base_E_sk_metric` for?

Reply:
that is for testing when the `base` parameter in our own implementation is different from log-base-10 (in this case log-e, i.e. the natural logarithm)